.circleci/config.yml

```diff
@@ -616,9 +616,9 @@ jobs:
           <<: *generate_protobufs
 
       - run:
-          name: Run make develop
+          name: Run make init develop
           command: |
-            make develop
+            make init develop
```

Comment:
`make init` must be run to install pipenv dependencies (without this, `make develop` creates a venv that's missing a bunch of streamlit dependencies)